tunnel/IpTunnel_admin.c

```diff
@@ -24,6 +24,9 @@
 #include "util/platform/Sockaddr.h"
 
 #include <stddef.h>
+#include <string.h>
```

Comment:
importing string.h makes me a bit twitchy because number 1 there are really bad functions like strdup() which need to be avoided and 2 there are functions which are subtly different on microsoft because fuck microsoft...  While you *could* implement parseIPandPrefix() with stuff in util/CString.h and util/Base10.h, I'd prefer that routingPrefix be a second (optional) argument in the API.